### PR TITLE
change scheduler.get_last_lr to get_lr to avoid bug

### PR DESCRIPTION
**Pull request by TobiasLee**
Learning rate scheduler function for getting last lr is `get_lr` instead of `get_last_lr`.

### CODE REVIEW

**Comment by julien-c** [comment]
I don’t think that’s right. Which version of PyTorch are you running?

**Comment by TobiasLee** [comment]
> I don’t think that’s right. Which version of PyTorch are you running?

Well, my version is 1.2.0, I checked the API in 1.4.0, the current version is right. 
Sorry for my hurry PR,
